app/lib/response_util.py

```diff
@@ -1,3 +1,16 @@
+def _returnUser(user):
```

Comment:
So we can extend our `User` model to have any methods we want. For example, we could give it a `toString` method that returns `I am a user with name % self.name`. Does that give you any ideas about how we can push this logic into the User model? Also, I can't believe I didn't think about this earlier. Sorry!